internal/controller/components/kserve/kserve.go

```diff
@@ -25,11 +24,9 @@ import (
 
 const (
 	componentName            = componentApi.KserveComponentName
-	authorinoOperator        = "authorino-operator"
-	serviceMeshOperator      = "servicemeshoperator"
-	serverlessOperator       = "serverless-operator"
 	kserveConfigMapName      = "inferenceservice-config"
 	kserveManifestSourcePath = "overlays/odh"
+	serviceMeshV2Operator    = "servicemeshoperator.v2"
```

Comment:
Not sure if it makes sense, checking if ServiceMesh v2 is there. ServiceMesh v3 will be installed already in the cluster.